src/xdg-open.c

```diff
@@ -45,6 +48,8 @@ main (int argc, char *argv[])
   GError *error = NULL;
   GDBusConnection *connection;
   GVariantBuilder opt_builder;
+  GFile *file;
+  char *uri;
```

Comment:
Unused variable - either remove it, or use

  const char *uri = uris[0];

to keep the code that ignores any uri but the first one in a single place ...